src/components/Form.jsx

```diff
@@ -74,4 +77,189 @@ export const AcceptanceForm = ({request, handleClose}) => {
 
 }
 
+const TimeSelector = ({setTimer}) => {
+    const [selectedDay, setSelectedDay] = useState('');
+    const [selectedHour, setSelectedHour] = useState('');
+    const [selectedMinute, setSelectedMinute] = useState('');
+
+    const days = Array.from({ length: 8 }, (_, i) => i); // Generates array [1, 2, 3, ..., 7]
+    const hours = Array.from({ length: 24 }, (_, i) => i);  // Generates array [0, 1, 2, ..., 23]
+    const minutes = Array.from({ length: 60 }, (_, i) => i);  // Generates array [0, 1, 2, ..., 59]
+
```

Comment:
I'm confused about the pre-defined `days`, `hours`, and `minutes`. Are we restricting the user  to only these values? I may have missed it